docs/coding-standards/tests/e2e/common-rule.md

```
@@ -0,0 +1,9 @@
+# E2Eテスト共通ルール
+
+## ディレクトリ構造
+E2Eテストは機能ごとに作成し、docs/designのディレクトリ構造を踏襲した形で配置する。
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)